template/build/webpack.prod.conf.js

```diff
@@ -70,6 +71,26 @@ module.exports = merge(baseWebpackConfig, {
     new webpack.optimize.CommonsChunkPlugin({
       name: 'manifest',
       chunks: ['vendor']
+    }),
+    // auto generate service-worker for caching static files
+    new SWPrecachePlugin({
+      cacheId: 'quasar-app',
+      filename: 'service-worker-cache.js',
+      minify: true,
+
+      staticFileGlobs: [
+        'js/**.js',
+        '/**.css'
+      ],
+
+      runtimeCaching: [{
+        urlPattern: /\/.*/,
+        handler: 'networkFirst'
```

Comment:
Why did you choose `networkFirst`? I would prefer using the cached copies first, then refetching the changed files and using them on the next reload. If I understand correctly `networkFirst` would refetch the unchanged files on all refresh which would be suboptimal on mobile internet, right?